spec/support/settings_helper.rb

```diff
@@ -17,3 +17,7 @@ def stub_server_configuration(config, config_name = "vmdb")
   configuration = double(:config => config.deep_symbolize_keys)
   allow(VMDB::Config).to receive(:new).with(config_name) { configuration }
 end
+
+def stub_server_settings(top_key, value)
+  allow(::Settings).to receive(top_key).and_return(value)
```

Comment:
I don't like this as it doesn't enforce that value is a Config::Options, so you are forced to write the code to access like a Hash, instead of with dot access as you'd expect.  Prefer the way it's done in `stub_settings`